coops-ui/src/main.ts

```diff
@@ -4,23 +4,28 @@ import '@mdi/font/css/materialdesignicons.min.css' // ensure you are using css-l
 import Vue from 'vue'
 import Vuetify from 'vuetify'
 import 'vuetify/dist/vuetify.min.css'
-import App from '@/App.vue'
 import Vuelidate from 'vuelidate'
 import Vue2Filters from 'vue2-filters'
 import Affix from 'vue-affix'
+import configHelper from '@/utils/config-helper'
 import router from '@/router'
 import store from '@/store/store'
-import configHelper from '@/utils/config-helper'
-import '@/registerServiceWorker'
 import { withFlagProvider } from 'ld-vue'
+import '@/registerServiceWorker'
+import '@/assets/styles/base.scss'
+import '@/assets/styles/layout.scss'
+import '@/assets/styles/overrides.scss'
```

Comment:
Have you considered creating an Index for the various scss files to help minimize/clean up the imports?

Reply:
It's probably not a good idea in this case because this is the only place we want to import all of them (and in this specific order). There's also a 4th scss file that we import by itself in many other places. I think it's safer this way in this case.